pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -69,6 +72,22 @@ def forward(self, image: torch.Tensor) -> torch.Tensor:
         )
 
 
+def license_info(repository: str = "ulyanov") -> str:
```

Comment:
Same as above for Johnson. Can't we have two constants like `LICENSE_JOHNSON` and `LICENSE_ULYANOV`?